TestFoundation/TestNSDictionary.swift

```diff
@@ -217,6 +219,19 @@ class TestNSDictionary : XCTestCase {
         // has crashed in the past
         dict["stringKey"] = "value"
     }
+    
+    func test_valueForKey() {
+        let dict: NSDictionary = ["foo": "bar"]
+        let result = dict.value(forKey: "foo")
+        XCTAssert(result as? String == "bar")
+    }
```

Comment:
Not a bit issue buts its generally better to use `XCTAssertEqual()` as it shows the incorrect value if the test fails.